### PR TITLE
prep release 17.2.2

### PR DESCRIPTION
**Pull request by jensengar**

### CODE REVIEW

**Review by zservies — APPROVED** [review]

**Review by hjon — APPROVED** [review]

**Comment by ben940531** [comment]
Hi, are you planning to merge this PR anytime soon? We have a snyk issue coming from this repo and it's blocking our deployments. We have the option to ignore snyk issue until this PR is merged, or wait for the merge if it'll come soon.

**Review by RobinsonWM — APPROVED** [review]

**Pull request merged**
